content/oci/oci.go

```diff
@@ -192,6 +193,44 @@ func (s *Store) Resolve(ctx context.Context, reference string) (ocispec.Descript
 	return desc, nil
 }
 
+// Untag removes a reference string from index.
+// reference should be a valid tag (e.g. "latest").
+// Reference: https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/image-layout.md#indexjson-file
+func (s *Store) Untag(ctx context.Context, descr ocispec.Descriptor, reference string) error {
```

Comment:
What's the purpose and scenario for `Untag`? If it is not related to this PR, could you create a new issue / PR accordingly?

Reply:
Hi @shizhMSFT,

The scenario where we want to use Untag, in our use case happens when a user has the same image with two different tags and want to remove one from the local OCI store. 

In essence we want to be able to keep the same manifest, config and layers with different references and be able to remove one of them if desired. 

Do you think this would be a valid use-case scenario for you ? 

I will gladly extract this in a separate PR if it makes sense to be on its own.

Reply:
I agree that this deserves to be its own PR. It's a valid use case, but not necessary to implement deletion.